### PR TITLE
Automatically update website when creating new release

### PR DESCRIPTION
**Pull request by Miepee**
I tested parts of it in https://github.com/Miepee/randovania.github.io where it did work

### CODE REVIEW

**Review by duncathan — APPROVED** [review]
lgtm for the website side

**Comment by Miepee** [comment]
noticed a few dumb things with it, currently fixing

**Comment by Miepee** [comment]
should hopefully:tm: work now.

File: .github/workflows/build-test-publish.yml [hunk 1]
```diff
@@ -621,3 +621,39 @@ jobs:
           env: ${{ steps.deployment.outputs.env }}
           deployment_id: ${{ steps.deployment.outputs.deployment_id }}
           override: false
+
+
+  update-website:
+    runs-on: ubuntu-latest
+    name: Update the website
+    if: ${{ startsWith(github.ref, 'refs/tags/v') }}
+    needs: release
```
Comment (henriquegemignani) [inline .github/workflows/build-test-publish.yml:630]: Can you add this block
```
    needs: release

    permissions:
      contents: write

    steps:
```

**Review by henriquegemignani — CHANGES_REQUESTED** [review]

File: .github/workflows/build-test-publish.yml (hunk 1 above)
Comment (Miepee) [reply .github/workflows/build-test-publish.yml:630]: done

**Review by henriquegemignani — APPROVED** [review]

**Pull request merged**